variables.tf

```diff
@@ -1,5 +1,22 @@
-variable "name" {
-  type = string
-  default = "test"
+variable "prefix" {  
 }
 
+variable "resource_group_name" {
+}
+
+variable "address" {  
+}
+
+variable "subnets" {    
+}
+
+variable "newbits" {
+  default = 4
+}
+
+variable "service_endpoints" {  
+}
+
+variable "tags" {
```

Comment:
Does tags need to be required? Can we default to empty if not please.

Reply:
yes, it is linked on the resource, just not set as anything.

Reply:
fixed; will obviously break, if the rg doesnt have the tags ;)